### PR TITLE
Switching MySQL Layers to DynamoDB

### PR DESCRIPTION
**Pull request by abdullah5abid**

### CODE REVIEW

**Review by tonytan4ever — COMMENTED** [review]
1. Can you make the work generic enough to accommodate both DynamoDB and SQL ?
2.  Update docker file, to use [localstack](https://localstack.cloud/) to test out if DynamoDB is working.

File: src/migrations/1665473850012-AddUserEntity.ts
Comment (tonytan4ever) [inline src/migrations/1665473850012-AddUserEntity.ts:1]: Please keep the original SQL migration.
Change config to use DynamoDB in a different folder

Comment (abdullah5abid) [reply src/migrations/1665473850012-AddUserEntity.ts:1]: Okay

Comment (tonytan4ever) [reply src/migrations/1665473850012-AddUserEntity.ts:1]: I didn't see this comments addressed.
Also please update README file and the [package.json](https://github.com/Carma-tech/CarmaTech-API/blob/main/package.json#L7) file to reflect the commands used to switch between SQL/Dynamo db

**Pull request closed without merging**